tidb-cloud/serverless-high-availability.md

```diff
@@ -0,0 +1,118 @@
+
+When you create a cluster with the default zonal high availability, all components, including Gateway, TiDB, TiKV, and TiFlash compute/write nodes, run in the same availability zone. The placement of these components in the data plane offer infrastructure redundancy with virtual machine pools, which minimizes failover time and network latency due to colocation.
+
+![TiDB Cloud Serverless zonal high availability](/media/tidb-cloud/serverless-zonal-high-avaliability-aws.png)
+
+In zonal high availability architecture:
+
+- The Placement Driver (PD) is deployed across multiple availability zones, ensuring high availability by replicating data redundantly across zones.
+- Data is replicated across TiKV servers and TiFlash write nodes within the local availability zone.
+- TiDB servers and TiFlash compute nodes read from and write to TiKV and TiFlash write nodes, which are safeguarded by storage-level replication.
+
+### Failover process
+
+TiDB Cloud Serverless ensures a transparent failover process for your applications. During a failover:
+
+- A new replica is created to replace the failed one.
+
+- Servers providing storage services recover local caches from persisted data on Amazon S3, restoring the system to a consistent state with the replicas.
+
+In the storage layer, persisted data is regularly pushed to Amazon S3 for high durability. Moreover, immediate updates are not only replicated across multiple TiKV servers but also stored on the EBS of each server, which further replicates the data for additional durability. TiDB automatically resolves issues by backing off and retrying in milliseconds, ensuring the failover process remains seamless for client applications.
+
+The gateway and computing layers are stateless, so failover involves restarting them elsewhere immediately. Applications should implement retry logic for their connections. While the zonal setup provides high availability, it cannot handle an entire zone failure. If the zone becomes unavailable, downtime will occur until the zone and its dependent services are restored.
+
+## Regional high availability architecture
+
+When you create a cluster with regional high availability, critical OLTP (Online Transactional Processing) workload components, such as PD and TiKV, are deployed across multiple availability zones to ensure redundant replication and maximizing availability. During normal operations, components like Gateway, TiDB, and TiFlash compute/write nodes are hosted in the primary availability zone. These components in data plane offer infrastructure redundancy through virtual machine pools, which minimizes failover time and network latency due to colocation.
+
+> **Note:**
+>
+> - Regional high availability is currently in beta and only available in the AWS Tokyo (`ap-northeast-1`) region.
+> - You can enable regional high availability only during cluster creation.
```

Comment:
Are there any UI changes for the regional HA? If yes, we also need to update this doc https://docs.pingcap.com/tidbcloud/create-tidb-cluster-serverless